tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/VersionMismatchTests.cs

```diff
@@ -0,0 +1,148 @@
+                {
+                    e.Value.Response = new MockTracerResponse("""{"data":{"id":"b5a855bffe6c0b2ae5d150fb6ad674363464c816","type":"ci_app_tracers_test_service_settings","attributes":{"code_coverage":false,"efd_enabled":false,"flaky_test_retries_enabled":false,"itr_enabled":true,"require_git":false,"tests_skipping":true}}} """, 200);
+                    return;
+                }
+
+                if (e.Value.PathAndQuery.EndsWith("api/v2/ci/tests/skippable"))
+                {
+                    e.Value.Response = new MockTracerResponse($"{{\"data\":[],\"meta\":{{\"correlation_id\":\"{correlationId}\"}}}}", 200);
+                    return;
+                }
+
+                if (e.Value.PathAndQuery.EndsWith("api/v2/citestcycle"))
+                {
+                    var lstEvents = new List<string>();
+                    var payload = JsonConvert.DeserializeObject<MockCIVisibilityProtocol>(e.Value.BodyInJson);
+                    if (payload.Events?.Length > 0)
+                    {
+                        foreach (var @event in payload.Events)
+                        {
+                            if (@event.Content.ToString() is { } eventContent)
+                            {
+                                if (@event.Type == SpanTypes.Test)
+                                {
+                                    tests.Add(JsonConvert.DeserializeObject<MockCIVisibilityTest>(eventContent));
+                                }
+                                else if (@event.Type == SpanTypes.TestSuite)
+                                {
+                                    testSuites.Add(JsonConvert.DeserializeObject<MockCIVisibilityTestSuite>(eventContent));
+                                }
+                                else if (@event.Type == SpanTypes.TestModule)
+                                {
+                                    testModules.Add(JsonConvert.DeserializeObject<MockCIVisibilityTestModule>(eventContent));
+                                }
+                                else
+                                {
+                                    customSpans.Add(JsonConvert.DeserializeObject<MockSpan>(eventContent));
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            using var processResult = await RunDotnetTestSampleAndWaitForExit(
+                                          agent,
+                                          arguments: "--collect:\"XPlat Code Coverage\"");
+
+            Assert.Equal(0, processResult.ExitCode);
+
+            // Check the tests, suites and modules count
+            Assert.Single(tests);
+            Assert.Single(testSuites);
+            Assert.Single(testModules);
+
+            // Check the custom spans count comming from mismatched version of Datadog.Trace
+            Assert.Single(customSpans);
```

Comment:
nit
```suggestion
            processResult.ExitCode.Should().Be(0);

            // Check the tests, suites and modules count
            tests.Should().ContainSingle();
            testSuites.Should().ContainSingle();
            testModules.Should().ContainSingle();

            // Check the custom spans count comming from mismatched version of Datadog.Trace
            customSpans.Should().ContainSingle();
```

Reply:
done in [f337ea8](https://github.com/DataDog/dd-trace-dotnet/pull/5578/commits/f337ea8c318021939998571bf8f682e8012b8257)